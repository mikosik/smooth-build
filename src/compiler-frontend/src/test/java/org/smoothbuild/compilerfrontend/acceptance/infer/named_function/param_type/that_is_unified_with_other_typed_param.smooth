myFunc(Int x, y) = [x, y];
