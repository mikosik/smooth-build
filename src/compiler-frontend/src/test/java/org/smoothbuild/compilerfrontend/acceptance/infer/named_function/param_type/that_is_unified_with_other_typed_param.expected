[Int] myFunc<>(Int x, Int y)
  = [x, y];
